tests/Resources/ti.media.audioplayer.addontest.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.Media', function () {
+	it('#createAudioPlayer()', function () {
+		should(Ti.Media.createAudioPlayer).be.a.Function;
+	});
+});
+
+describe('Titanium.Media.AudioPlayer', function () {
+	var audioPlayer;
+
+	beforeEach(function () {
+		audioPlayer = Ti.Media.createAudioPlayer({ url: 'sample.mp3' });
```

Comment:
Can you change `'sample.mp3'` to `'/sample.mp3'` please?

Android does not currently support loading APK asset directory files via relative paths yet until [TIMOB-13642] has been fixed. Will avoid a unit test failure.